src/main/resources/xml/schema/2020-1/nordic-html5.rng

```diff
@@ -2095,22 +2095,29 @@
             <attribute name="class">
                 <value>verse</value>
             </attribute>
-            <optional>
+            <zeroOrMore>
                 <ref name="bridgehead"/>
-            </optional>
+                <ref name="poem.author"/>
+            </zeroOrMore>
             <oneOrMore>
                 <ref name="poem.linegroup"/>
             </oneOrMore>
-            <optional>
-                <element name="p">
-                    <attribute name="class">
-                        <value>verse-author</value>
-                    </attribute>
-                </element>
-            </optional>
+            <zeroOrMore>
+                <ref name="bridgehead"/>
+                <ref name="poem.author"/>
+            </zeroOrMore>
```

Comment:
I think `<ref name="poem.linegroup"/>` is needed inside this `<zeroOrMore>` to allow for for instance bridgeheads between linegroups.

Reply:
Great catch, had to think about it a bit but just adding it to the last clause would be ok.